datalake-dr-connector/src/main/java/com/sequenceiq/cloudbreak/datalakedr/config/DatalakeDrConfig.java

```diff
@@ -51,7 +66,15 @@ public int getPort() {
         return port;
     }
 
-    public boolean isConfigured() {
-        return enabled && StringUtils.isNotBlank(endpoint);
+    public String getMockHost() {
+        return mockHost;
+    }
+
+    public int getMockPort() {
+        return mockPort;
+    }
+
+    public boolean isConfigured(boolean mockCloud) {
```

Comment:
We can overload the `isConfigured()` method to provide an "old" solution that can call the new one. in that case, you do not need to change all possible locations of the usage of this method.

e.q:
```java 
public boolean isConfigured(){
   return isConfigured(true);
}
```

Reply:
Yeah this may be cleaner thank you.

Reply:
Actually looking at this again, this would only be called in 3 places with 2 of those being tests so I don't think it's necessary.

Reply:
sorry, in that case just ignore my comments!